### DISCUSSION TITLE
TP3 chequeo ejercicio 1

### DISCUSSION THREAD

**Discussion by Juampi44** [INGCOM-UNRN-P1/cursada-2023#111] [Preguntas y respuestas] [answered]
queria saber si voy bien encaminado en el ejercicio 1 https://github.com/INGCOM-UNRN-P1/tp3-2023-Juampi44

**Comment by martinvilu** [comment, marked as answer]
La capacidad es fija, no a partir de una variable.

El código del main, debiera de ir en función/es.

Eso visto por arriba...

**Reply by Juampi44** [reply]
ok, gracias

**Reply by martinvilu** [reply]
Alguna duda?

**Reply by Juampi44** [reply]
Nop, todo bien, gracias por revisar como venía
